workers/unity/Assets/Gdk/Core/Components/WorldCommandsTranslation.cs

```diff
@@ -184,7 +196,18 @@ public class WorldCommandsTranslation : ComponentTranslation
             typeof(CommandResponses<EntityQueryResponse>),
         };
 
-        private Dictionary<uint, long> RequestIdToEntityId = new Dictionary<uint, long>();
+        private readonly Dictionary<uint, CreateEntityRequest> requestIdToCreateEntityRequest
+            = new Dictionary<uint, CreateEntityRequest>();
+
+        private Dictionary<uint, DeleteEntityRequest> requestIdToDeleteEntityRequest
+            = new Dictionary<uint, DeleteEntityRequest>();
+
+        private Dictionary<uint, ReserveEntityIdsRequest> requestIdToReserveEntityIdsRequest
+            = new Dictionary<uint, ReserveEntityIdsRequest>();
+
+        private Dictionary<uint, EntityQueryRequest> requestIdToEntityQueryRequest
+            = new Dictionary<uint, EntityQueryRequest>();
+        // Also for other 3!
```

Comment:
Can you further qualify or remove this comment?

Reply:
Also result of rushing, I put that there before adding all 4 dictionaries. Note to self: read through the entire diff next time.